plugins/woocommerce/src/Caching/WPCacheEngine.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Automattic\WooCommerce\Caching;
+
+/**
+ * Implementation of CacheEngine that uses the built-in WordPress cache.
+ */
+class WPCacheEngine implements CacheEngine {
+	use CacheNameSpaceTrait;
+
+	/**
+	 * Retrieves an object cached under a given key.
+	 *
+	 * @param string $key They key under which the object to retrieve is cached.
+	 * @param string $group The group under which the object is cached.
+	 *
+	 * @return array|object|null The cached object, or null if there's no object cached under the passed key.
+	 */
+	public function get_cached_object( string $key, string $group = '' ) {
```

Comment:
Is it possible there is user code out there (perhaps within some sites running on WP Engine) that extends this class/is it safe to change the method in this way (ditto some of the other methods where we change the method signature)?

Reply:
Just linking that it was answered here.